### DISCUSSION TITLE
how do I change default 16 horizontal opened sessions by hovering  on kitty icon ?

### DISCUSSION THREAD

**Discussion by tjyang** [kovidgoyal/kitty#6942] [Q&A]
- When one move the mouse to kitty icon on task bar
![image](https://github.com/kovidgoyal/kitty/assets/378638/5e0cc16e-9720-4e2b-84df-e170bfd4786b)

- this is my desired action to list out opened sessions vertically. but this behaviour only happened if you have 17+ sessions.
![image](https://github.com/kovidgoyal/kitty/assets/378638/23c50064-c96c-42e2-89f1-26d182055c4c)

- Question: How do I change the default 16 sessions for displaying horizontally to just said 3 sessions ?

**Comment by kovidgoyal** [comment]
see https://github.com/cyd01/KiTTY
